bin/host/src/interop/handler.rs

```diff
@@ -36,7 +36,7 @@ use kona_protocol::BlockInfo;
 use kona_registry::ROLLUP_CONFIGS;
 use std::sync::Arc;
 use tokio::task;
-use tracing::warn;
+use tracing::{debug, warn};
```

Comment:
FWIW, don't need to have these imports if you do this in the top-level `lib.rs`

```
#[macro_use]
extern crate tracing;
```

Reply:
will follow up - agree this is cleaner.